src/views/Nft/market/Collection/IndividualNFTPage/MoreFromThisCollection.tsx

```diff
@@ -97,9 +103,11 @@ const MoreFromThisCollection: React.FC<MoreFromThisCollectionProps> = ({ collect
 
   return (
     <Box pt="56px" mb="52px">
-      <Text bold mb="24px">
-        {t('More from this collection')}
-      </Text>
+      {title && (
+        <Text bold mb="24px">
+          {t(title)}
```

Comment:
Should it be title? 🤔 
And even if it is better - probably no need to `t()` it - I don't think collection names are translatable?

Reply:
Also title will be present all the time since it has default value, no need to `&&` it

Reply:
Title is not the collection name here. The component was always displaying `More from this collection` above the slider.
I made it a prop so the component can be more reusable.